apisix/plugins/kafka-logger.lua

```diff
@@ -68,6 +68,19 @@ local schema = {
                         maximum = 65535,
                         description = "the port of kafka broker",
                     },
+                    sasl_config = {
+                        type = "object",
+                        description = "sasl config",
+                        properties = {
+                            mechanism = {
+                                type = "string",
+                                default = "PLAIN"
```

Comment:
We can use enum to make sure the passing mechanism is legal? Although only `PLAIN` is currently supported.

Reply:
Good suggestion. Let me add the enum and add a test case to cover it.